src/main/java/nl/dtls/fairdatapoint/config/TemplateConfig.java

```diff
@@ -109,7 +109,7 @@ private static String literal(Literal literal, Options options) {
      */
     private static String simplifyUrl(IRI iri, Options options) {
         if (iri != null) {
-            String[] parts = iri.toString().split("/");
+            String[] parts = iri.toString().split("[/#:]");
             if (parts.length > 0) {
                 return parts[parts.length - 1];
             }
```

Comment:
You could use [`IRI.getLocalName()`](https://rdf4j.eclipse.org/javadoc/latest/org/eclipse/rdf4j/model/IRI.html#getLocalName--), which would reduce the complexity.